react-accordion-component/src/accordian.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+
+export default function Accordian({ items }) {
+
+  return (
+    <div className='row'>
+      {items.map((item) => <ListItem key={item.language} item={item} />)}
+    </div>
+  );
+}
+
+function ListItem({ item }) {
```

Comment:
This implementation allows multiple items to be opened at the same time. The spec says that is not allowed. You need to pull the "isOpen" state up to its parent to get it to work properly.